gbpservice/nfp/configurator/config_files/asav.conf

```diff
@@ -0,0 +1,22 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+[ASAV_FW_CONFIG]
+
+mgmt_username = admin
+mgmt_userpass = b2Nhc2F2dm0=
```

Comment:
We need to figure out how to get rid of passwords in config files. Lets put a TODO for this.